frontera/contrib/messagebus/zeromq/broker.py

```diff
@@ -107,23 +108,23 @@ def handle_db_out_recv(self, msg):
 
     def handle_db_in_recv(self, msg):
         self.stats['db_in_recvd'] += 1
-        if msg[0][0] in ['\x01', '\x00']:
+        if to_native_str(msg[0])[0] in ['\x01', '\x00']:
```

Comment:
We can use - `if b'\x01' in msg[0] or b'\x00' in msg[0]:`. The other option would be for PY2 to use `msg[0][0]` as is(because it will be a byte) and for PY3 convert `msg[0][0]` to character code(from int).